### DISCUSSION TITLE
OBS Websocket not connecting to TriggerFyre?

### DISCUSSION THREAD

**Discussion by PinkVomit96** [Palakis/obs-websocket#911] [Q&A]
I'm not tech savvy so forgive me if I'm being dumb and missing something. I was following a tutorial on how to install triggerfyre, and went to download the websocket to make that happen. however no matter how many times I've changed the passwords, uninstalled and reinstalled the websocket and OBS, it just won't connect. It instantly rejected it the first time, but then i uninstalled, reinstalled, and changed the password. Now it just sits there 'loading' but not connecting, no matter how long i leave it. I've tried multiple different passwords but nothings happening. I don't know if this means it's an issue with triggerfyre or the websocket? I'm at a loss :(

 Any advice would be really appreciated! TYIA

**Comment by tt2468** [comment]
Make sure that you have downloaded version 4.9.1 and not a 5.x version. 5.x is beta and not compatible with 4.x

**Reply by PinkVomit96** [reply]
yep definitely did that :(
